{"uid":"c950d222e34332b9","name":"test_network_cut_ip_change_application_https","fullName":"tests.integration.ha.test_network_cut#test_network_cut_ip_change_application_https","historyId":"4a7dbe428fbcaf0701b5a9b0ec1a184e","time":{"start":1727403393849,"stop":1727406822324,"duration":3428475},"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 848, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 879, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/tests/integration/ha/test_network_cut.py\", line 420, in test_network_cut_ip_change_application_https\n    await network_cut_application(ops_test, https=True)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/tests/integration/ha/test_network_cut.py\", line 261, in network_cut_application\n    await ops_test.model.block_until(\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1046, in block_until\n    await utils.block_until(done,\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1727400608098,"stop":1727400608098,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"opensearch_sysctl_settings","time":{"start":1727400608098,"stop":1727400608130,"duration":32},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1727400608098,"stop":1727400608098,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1727400608130,"stop":1727400608273,"duration":143},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1727403393849,"stop":1727403393849,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 129, in _block\n    await jasyncio.sleep(wait_period)\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 605, in sleep\n    return await future\nasyncio.exceptions.CancelledError\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 456, in wait_for\n    return fut.result()\nasyncio.exceptions.CancelledError\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 848, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 831, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 879, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/tests/integration/ha/test_network_cut.py\", line 420, in test_network_cut_ip_change_application_https\n    await network_cut_application(ops_test, https=True)\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/tests/integration/ha/test_network_cut.py\", line 261, in network_cut_application\n    await ops_test.model.block_until(\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 1046, in block_until\n    await utils.block_until(done,\n  File \"/home/ubuntu/actions-runner/_work/opensearch-dashboards-operator/opensearch-dashboards-operator/.tox/integration/lib/python3.10/site-packages/juju/utils.py\", line 130, in block_until\n    await jasyncio.shield(jasyncio.wait_for(_block(), timeout))\n  File \"/usr/lib/python3.10/asyncio/tasks.py\", line 458, in wait_for\n    raise exceptions.TimeoutError() from exc\nasyncio.exceptions.TimeoutError","steps":[],"attachments":[{"uid":"3e07ed0939eb0283","name":"log","source":"3e07ed0939eb0283.txt","type":"text/plain","size":1354},{"uid":"a0a8f2e1a103b627","name":"stdout","source":"a0a8f2e1a103b627.txt","type":"text/plain","size":148},{"uid":"8525e378845607f6","name":"stderr","source":"8525e378845607f6.txt","type":"text/plain","size":460}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":3,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1727406822332,"stop":1727406822332,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1727406822336,"stop":1727406822768,"duration":432},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1727406822769,"stop":1727406822769,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1727406822769,"stop":1727406822769,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1727406822769,"stop":1727406822770,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha"},{"name":"suite","value":"test_network_cut"},{"name":"host","value":"github-runner"},{"name":"thread","value":"20665-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha.test_network_cut"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"343bb0823743a5b5","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":30,"passed":20,"unknown":1,"total":52},"items":[{"uid":"c0cfea108c3a8c4d","reportUrl":"../1165//#testresult/c0cfea108c3a8c4d","status":"passed","time":{"start":1727316990999,"stop":1727317741641,"duration":750642}},{"uid":"71e3b4ec610cb8f6","reportUrl":"../1158//#testresult/71e3b4ec610cb8f6","status":"passed","time":{"start":1727230565309,"stop":1727231346135,"duration":780826}},{"uid":"780845ee6629e377","reportUrl":"../1152//#testresult/780845ee6629e377","status":"passed","time":{"start":1727144245862,"stop":1727144936393,"duration":690531}},{"uid":"645d3d127d4f74b6","reportUrl":"../1141//#testresult/645d3d127d4f74b6","status":"passed","time":{"start":1727057757382,"stop":1727058569289,"duration":811907}},{"uid":"d24b157147b14372","reportUrl":"../1138//#testresult/d24b157147b14372","status":"passed","time":{"start":1726971550388,"stop":1726972239346,"duration":688958}},{"uid":"aa188a19844c157b","reportUrl":"../1137//#testresult/aa188a19844c157b","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726887211878,"stop":1726887211878,"duration":0}},{"uid":"f8c4ba0a311559d2","reportUrl":"../1131//#testresult/f8c4ba0a311559d2","status":"passed","time":{"start":1726799671274,"stop":1726800362212,"duration":690938}},{"uid":"1faa9f797193fbde","reportUrl":"../1127//#testresult/1faa9f797193fbde","status":"passed","time":{"start":1726713441763,"stop":1726714132440,"duration":690677}},{"uid":"88843f54f76354f0","reportUrl":"../1126//#testresult/88843f54f76354f0","status":"passed","time":{"start":1726625856746,"stop":1726626606775,"duration":750029}},{"uid":"6da16e5d706e2775","reportUrl":"../1124//#testresult/6da16e5d706e2775","status":"passed","time":{"start":1726539330874,"stop":1726540111533,"duration":780659}},{"uid":"a5c8473fb7902e3c","reportUrl":"../1122//#testresult/a5c8473fb7902e3c","status":"passed","time":{"start":1726453133250,"stop":1726453854138,"duration":720888}},{"uid":"40bd3f5cf1ff3ed1","reportUrl":"../1121//#testresult/40bd3f5cf1ff3ed1","status":"passed","time":{"start":1726366615681,"stop":1726367365740,"duration":750059}},{"uid":"dcf762caf9b32167","reportUrl":"../1120//#testresult/dcf762caf9b32167","status":"passed","time":{"start":1726280219562,"stop":1726280938722,"duration":719160}},{"uid":"44f61fae45fb16d0","reportUrl":"../1115//#testresult/44f61fae45fb16d0","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726193157459,"stop":1726193157459,"duration":0}},{"uid":"9e1714e20688d124","reportUrl":"../1103//#testresult/9e1714e20688d124","status":"passed","time":{"start":1726107416949,"stop":1726108197069,"duration":780120}},{"uid":"a3a0f60d2fda700a","reportUrl":"../1101//#testresult/a3a0f60d2fda700a","status":"passed","time":{"start":1726020976408,"stop":1726021756749,"duration":780341}},{"uid":"36fa6548723b117f","reportUrl":"../1094//#testresult/36fa6548723b117f","status":"passed","time":{"start":1725936044467,"stop":1725936856583,"duration":812116}},{"uid":"646dd4e0a0fc7b19","reportUrl":"../1089//#testresult/646dd4e0a0fc7b19","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1725850374232,"stop":1725850374232,"duration":0}},{"uid":"4e23229a8d99ce18","reportUrl":"../1088//#testresult/4e23229a8d99ce18","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1725762647455,"stop":1725762647455,"duration":0}},{"uid":"47a0c4ca0bb5a849","reportUrl":"../1087//#testresult/47a0c4ca0bb5a849","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1725674841471,"stop":1725674841471,"duration":0}}]},"tags":["runner(['self-hosted', 'linux', 'X64', 'jammy', 'large'])","asyncio","abort_on_fail","group(1)"]},"source":"c950d222e34332b9.json","parameterValues":[]}